packages/components/dashboard/details-card/details-card.tsx

```diff
@@ -0,0 +1,77 @@
+import * as React from 'react';
+import { useK8sWatchResource } from '@openshift-console/dynamic-plugin-sdk';
+import { HealthBody } from '@openshift-console/dynamic-plugin-sdk-internal';
+import * as _ from 'lodash';
+import { useTranslation } from 'react-i18next';
+import {
+  Card,
+  CardBody,
+  CardHeader,
+  CardTitle,
+  Gallery,
+  GalleryItem,
+  Skeleton,
+} from '@patternfly/react-core';
+import { DATA_FEDERATION_NAMESPACE, MCG_DEPLOYER } from '../../../constants';
+import { ClusterServiceVersionModel } from '../../../models';
+import { ClusterServiceVersionKind } from '../../../types';
+import { referenceForModel } from '../../../utils';
+import ResourceLink from '../../../utils/generics/resource-link';
+import { operatorResource } from '../../resources';
+import './details-card.scss';
+
+export const DetailsCard: React.FC = () => {
+  const { t } = useTranslation();
+  const [csvData, csvLoaded, csvError] =
+    useK8sWatchResource<ClusterServiceVersionKind[]>(operatorResource);
+  const operator = React.useMemo(
+    () => csvData?.find((csv) => csv?.metadata?.name.startsWith(MCG_DEPLOYER)),
+    [csvData]
+  );
+  const operatorVersion = operator?.spec?.version;
+  const operatorName = operator?.metadata?.name;
+  const path = `/k8s/ns/${DATA_FEDERATION_NAMESPACE}/${referenceForModel(
+    ClusterServiceVersionModel
+  )}/${operatorName}`;
+  return (
+    <Card>
+      <CardHeader>
+        <CardTitle>{t('Details')}</CardTitle>
+      </CardHeader>
+      <CardBody>
+        <HealthBody>
+          <Gallery
+            className="co-overview-status__health details-card-body"
+            hasGutter
+          >
+            <GalleryItem className="details-card-item">
+              <div className="details-card-sub-item">Operator name</div>
+              <div>
+                {operatorName ? (
+                  <ResourceLink
+                    resourceModel={ClusterServiceVersionModel}
+                    resourceName={t('Data Federation')}
+                    {...(csvLoaded && !csvError && { link: path })}
+                    hideIcon
+                  />
+                ) : (
+                  t('Data Federation')
+                )}
+              </div>
+            </GalleryItem>
+            <GalleryItem className="details-card-item">
+              <div className="details-card-sub-item">{t('Version')}</div>
+              {csvLoaded && !csvError ? (
```

Comment:
```suggestion
              {operatorVersion ? (
```

Reply:
since we don't have the CRs, it continues to show the skeleton.

![image](https://user-images.githubusercontent.com/22158580/173044645-6080350e-0e41-4d42-9f8b-62471246741d.png)


what to do in this scenario?

either we can show a small hypen '-' or no data available or even hide the version itself until we have any value to show for it